src/components/MonitoringPlanTabRender/MonitoringPlanTabRender.js

```diff
@@ -58,6 +58,42 @@ export const MonitoringPlanTabRender = ({
 
   const [revertedState, setRevertedState] = useState(false);
 
+  // Initialize state to collect data statuses
+  const [tableDataStatuses, setTableDataStatuses] = useState({});
+
+  const handleReportDataStatus = (tableName, dataStatus) => {
+    setTableDataStatuses((prevStatuses) => ({
+      ...prevStatuses,
+      [tableName]: dataStatus,
+    }));
+  };
```

Comment:
The `tableDataStatuses` currently maintains its values between sections, so this can cause an incorrect value for the _inactive_ checkbox. To prevent this, reset the `tableDataStatuses` state if the selected section changes:
```suggestion
  };
  
  const [prevSectionSelect, setPrevSectionSelect] = useState(sectionSelect);
  if (prevSectionSelect !== sectionSelect) {
    setPrevSectionSelect(sectionSelect);
    setTableDataStatuses({});
  }
```